esp/src/tests/v9-workunits.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { test, expect } from "@playwright/test";
+
+test.describe("V9 Workunits", () => {
+
+    test.beforeEach(async ({ page }) => {
+        await page.goto("index.html#/workunits");
+        await page.waitForLoadState("networkidle");
+    });
+
+    test("Loaded", async ({ page }) => {
```

Comment:
I'd argue for more descriptive titles for all the tests. I suppose (more often than not) no one will look at these... But when anyone does, something like "WU list loaded" vs "Loaded"?